src/main/java/seedu/address/logic/commands/AddRecruitmentPostCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_POST;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+
+/**
+ * Adds a recruitment post on the address book.
+ */
+public class AddRecruitmentPostCommand extends Command {
+
+    public static final String COMMAND_WORD = "recruitmentPost";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Job Position "
+            + PREFIX_POST;
```

Comment:
You should spend some time to think about your full string of prefix(es) alongside some example after each prefix